lib/srv/desktop/windows_server.go

```diff
@@ -936,7 +936,7 @@ func (s *WindowsService) makeTDPSendHandler(ctx context.Context, emitter events.
 ) func(m tdp.Message, b []byte) {
 	return func(m tdp.Message, b []byte) {
 		switch b[0] {
-		case byte(tdp.TypePNG2Frame), byte(tdp.TypePNGFrame), byte(tdp.TypeError), byte(tdp.TypeNotification):
+		case byte(tdp.TypeRDPFastPathPDU), byte(tdp.TypePNG2Frame), byte(tdp.TypePNGFrame), byte(tdp.TypeError), byte(tdp.TypeNotification):
```

Comment:
How does the size of this PDU compared to the bitmap PDUs we're accustomed to?

Reply:
I experimented by adding some logging here and then: running a desktop session, opening an edge browser window and scrolling up and down on the default page. The stats are for the `len(b)`

#### `tdp.TypePNG2Frame`

- min length: 460
- max length: 15395
- mean length: 1467.018153

#### `tdp.TypeRDPFastPathPDU`
- min length: 10
- max length: 16236
- mean length: 10595.500000

So they appear comparable, at least for basic activity.

Reply:
Double checking with playing a youtube video:

#### `tdp.TypePNG2Frame`
- min length: 460
- max length: 14683
- mean length: 1682.348999

#### `tdp. TypeRDPFastPathPDU `
- min length: 10 
- max length: 16236
- mean length: 12887.359416